README.md

```diff
@@ -26,10 +26,13 @@ _To adapt for your language (the basics):_
 
 For more information, including how to localize the interface to your language, visit the blog article: https://blog.mothertongues.org/wordle/.
 
-The interface is translated by default in both English and Spanish - other translations are very welcome! 
+The interface is translated by default in English, Kiswahili, Mandarin and Spanish - other translations are very welcome!  These translations require
+1. Adding an appropriate localiztion file in `public/locales`
```

Comment:
Yes - thanks for adding these instructions! I might also add the the language code to the config.ts file too by default.